cms/templates/ux/reference/container.html

```diff
@@ -151,7 +151,15 @@ <h3 class="sr">Page Actions</h3>
                                                                             <iframe id="i4x-AndyA-ABT101-video-72b5a0d74e8c4ed4a4d4e6bf67837c09" frameborder="0" allowfullscreen="1" title="YouTube video player" width="640" height="360" src="https://www.youtube.com/embed/OEoXaMPEzfM?controls=0&amp;wmode=transparent&amp;rel=0&amp;showinfo=0&amp;enablejsapi=1&amp;modestbranding=1&amp;html5=1&amp;origin=http%3A%2F%2Flocalhost%3A8001" style="height: 411.75px; width: 732px; top: -22.875px; left: 0px;"></iframe>
                                                                             <h3 class="hidden">ERROR: No playable video sources found!</h3>
                                                                         </section>
+                                                                        <a href="#" tabindex="0" class="hide-captions" role="button" aria-disabled="false"
+                                                                              title="Turn off transcripts">
+                                                                          <i class="icon-caret-right" aria-hidden="true"></i>
```

Comment:
We've recently updated our Font Awesome library, so you'll want to update the icons to use the [latest Font Awesome set] (http://fontawesome.io/cheatsheet/). A rebase against Master might be wise too.